profile-widget/diveeventitem.cpp

```diff
@@ -152,17 +152,17 @@ void DiveEventItem::setupToolTipString()
 
 		/* Do we have an explicit cylinder index?  Show it. */
 		if (internalEvent->gas.index >= 0)
-			name += QString(" (cyl %1)").arg(internalEvent->gas.index+1);
+			name += tr(" (cyl. %1)").arg(internalEvent->gas.index + 1);
 	} else if (value) {
-		if (type == SAMPLE_EVENT_PO2 && name == "SP change") {
+		if (type == SAMPLE_EVENT_PO2 && (QString) internalEvent->name == "SP change") {
```

Comment:
we have a function same_string(const char *a, const char *b) that does that for ordinary C strings and that does not throw up for NULL pointers as strcmp() (the traditional solution). The == operator would compare the pointers and not the strings pointed to.